src/test/java/core/basesyntax/service/strategy/SubtractOperationHandlerTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SubtractOperationHandlerTest {
+    public static final Integer INITIAL_QUANTITY = 100;
+    public static final Transaction VALID =
```

Comment:
```suggestion
    public static final Transaction  valid =
```
якщо клас не immutable потрібно викорстовувати стандартниий неймінг